.env.development

```diff
@@ -1,6 +1,4 @@
-GRAVITY_API_BASE=http://localhost:3000/api/v1
-GRAVITY_GRAPHQL_ENDPOINT=http://localhost:3000/api
-GRAVITY_API_URL=http://localhost:3000
-#GRAVITY_ID=
-#GRAVITY_SECRET=
```

Comment:
Maybe we should leave these in. I'm realizing in situations where they don't want to sync data, we will need to include the values populated by `gravity`'s seed data.

Reply:
Another good callout. Added in https://github.com/artsy/metaphysics/pull/5420/commits/a32ffe82aedc9a599508ef4ab767db6062e81149

I assume we want those commented out by default?

Reply:
Yeah, I think so. Might be worth a note in the readme.